service/models.py

```diff
@@ -94,22 +78,62 @@ def init_db(cls, app):
 
     @classmethod
     def all(cls):
-        """ Returns all of the YourResourceModels in the database """
-        logger.info("Processing all YourResourceModels")
+        """ Returns all of the Wishlists in the database """
+        logger.info("Processing all Wishlists")
         return cls.query.all()
 
     @classmethod
     def find(cls, by_id):
-        """ Finds a YourResourceModel by it's ID """
+        """ Finds a Wishlist by it's ID """
         logger.info("Processing lookup for id %s ...", by_id)
         return cls.query.get(by_id)
 
-    @classmethod
-    def find_by_name(cls, name):
-        """Returns all YourResourceModels with the given name
+
+######################################################################
+#  W I S H L I S T   M O D E L
+######################################################################
+class Wishlist(db.Model, PersistentBase):
+    """
+    Class that represents a Wishlist
+    """
+
+    app = None
+
+    # Table Schema
+    wishlist_id = db.Column(db.Integer, primary_key=True)
```

Comment:
We might want to add some "sad path" tests for valid data types.  Like wishlist_names that are longer than 64 char.  (I confess I don't know yet whether Python and the db.Model class give us some basic validation when working with the model class instances themselves.

Reply:
I will write this up as a new story.